x/auth/client/rest/decode.go

```diff
@@ -55,6 +56,17 @@ func DecodeTxRequestHandlerFn(clientCtx client.Context) http.HandlerFunc {
 
 		response := DecodeResp(stdTx)
 
+		_, err = clientCtx.LegacyAmino.MarshalJSON(response)
+		if err != nil {
+			if strings.Contains(err.Error(), "unregistered concrete type") {
```

Comment:
We can factor this code into a function

Reply:
I believe the error messages are slightly different, but yes I like the idea.